app/src/main/kotlin/com/softteco/template/data/TemplateApi.kt

```diff
@@ -21,4 +23,7 @@ interface TemplateApi {
 
     @POST("/api/auth/registration")
     suspend fun registration(@Body user: CreateUserDto): String
+
+    @PATCH("/api/auth/resetPassword")
+    suspend fun resetPassword(@Body email: ResetPasswordDto): String
```

Comment:
The parameter encapsulates the password, not email

Reply:
fixed